tex/rules.tex

```diff
@@ -93,27 +93,38 @@
 by the RoboCupJunior Soccer Technical Committee. The English version of these
 rules has priority over any translations. \textcolor{red}{\textbf{Items in red
-represent modifications introduced this year.}}
+represent significant rules changes introduced this year.}}
 
 Teams are advised to check the RoboCupJunior Soccer site
 \href{http://rcj.robocup.org/soccer.html}{http://rcj.robocup.org/soccer.html}
-for OC (Organization Committee) procedures and requirements for the
-competition. Each team is responsible for verifying the latest version of the
-rules prior to the competition.
+for OC (Organizational Committee) procedures and requirements for the
+international competition. Each team is responsible for verifying the latest version of the
+rules prior to competition.
 
 \section*{Preface}
 
-In the RoboCupJunior soccer challenge, teams of two autonomous mobile robots
-compete against another team in matches. They must look for a ball, trying to
-score into a color-coded goal in a special field built in a way that resembles
-the actual field for human soccer. Robots are required to have full autonomy
-from humans, technical designs, and ingenious programming by their developers.
+In the RoboCupJunior soccer challenge, teams of young engineers design, build, 
+and program two fully autonomous mobile robots to compete against another team 
+in matches. The robots must detect a ball and score into a color-coded goal on 
+a special field that resembles a human soccer field.
 
-Participants of this challenge are required to give the best of their abilities
-in programming, robotics, electronics and mechatronics, but also to contribute
-on teamwork and knowledge sharing with other participants, regardless of
-culture, age or result in the competition. \textbf{All are expected to compete,
-learn, have fun and grow.}
+To be sucessessful, participants must demonstrate skill in programming, robotics, 
+electronics and mechatronics.  Teams are also expected to contribute to the
+the advancement of the community as a whole by sharing their discoveries with other
+participants and by engaging in good sportsmanship,regardless of culture, age or 
+result in the competition. \textbf{All are expected to compete,
+learn, have fun, and grow.}
+
+Robocup Junior Soccer consist of two sub-leagues:
+\textbf{``Soccer Open''} and \textbf{``Soccer Lightweight''}. These rules apply
+for both sub-leagues. There are two main differences between the two leagues.
+
+\textbf{``Soccer Lightweight''} is played using a special ball that emits an IR signal ball. Robots may weigh up to 1.1 kg, may have a ball-capturing zone of up to 2.5 cm, and may use batteries up to 15.0 nominal voltage.  
+
+\textbf{``Soccer Open''} is played using a passive, brighly colored orange ball. Robots may weigh up to 2.4 kg, may have a ball-capturing zone of up to 3.0 cm, and may use batteries up to 12.0 nominal voltage.  
+
+{please see 5. Ball, for balls specifications and 8. League
```

Comment:
@gngrbrdmn I am not sure what you wanted to do here, but it seems that "please" should probably be "Please" here.

Let me know if there is some formatting option you wanted to use.

Thanks!

Reply:
Yep.  Can you merge so that I can edit the file with the changes in it?

Reply:
@gngrbrdmn You should be able to continue editing the file with current changes in it, that is:

https://github.com/RoboCupJuniorTC/soccer-rules/blob/gngrbrdmn-patch-1/tex/rules.tex

I am also happy to merge it though -- if you edit the file above, the changes will stay in this Pull Request, which is something I would personally prefer =)